Makefile

```diff
@@ -32,12 +32,8 @@ _site build:
 clean:
 	docker run --rm -ti -e JEKYLL_UID=`id -u` -v $$PWD:/srv/jekyll jekyll/jekyll:$(JEKYLL_VERSION) jekyll clean
 
-htmlproofer: clean _site
-	docker run -ti -e JEKYLL_UID=`id -u` --rm -v $$PWD/_site:/_site/ quay.io/calico/htmlproofer /_site --file-ignore ${HP_IGNORE_LOCAL_DIRS} --assume-extension --check-html --empty-alt-ignore --url-ignore "/docs.openshift.org/,#,/github.com\/projectcalico\/calico\/releases\/download/,https://github.com/projectcalico/calico/tree/master/master/introduction/index.md,https://github.com/projectcalico/calico/tree/master/v2.6/introduction/index.md"
-	-docker run -ti -e JEKYLL_UID=`id -u` --rm -v $$PWD/_site:/_site/ quay.io/calico/htmlproofer /_site --assume-extension --check-html --empty-alt-ignore --url-ignore "#"
-	# Rerun htmlproofer across _all_ files, but ignore failure, allowing us to notice legacy docs issues without failing CI
-	
-	docker run -v $$PWD:/calico --entrypoint /bin/sh -ti garethr/kubeval:0.1.1 -c 'find /calico/_site/master -name "*.yaml" |grep -v config.yaml | xargs /kubeval'
+htmlproofer:
+	@echo "Do not make docs changes against this branch, please use master."
```

Comment:
What's the reasoning behind this?

Reply:
@ozdanborne this is because the htmlproofer checks bunch of github docs that no longer exist in the master branch because master is tracking v3.0.

The docs will always be rendered from the master branch anyway - this branch is just for maintaining a calico/node build, so we decided just to disable the htmlproofer check.

Reply:
Caseys comment from earlier:
>Given we're never building the docs from this branch - it's just for maintaining a v2.x calico/node >- we can probably just turn off htmlproofer entirely.
>
>i.e. there should never be docs PRs to this branch.

So I just replaced it with a message so we don't have to do anything clever in semaphore to git it working against this branch.